packages/simcore-sdk/src/simcore_sdk/node_ports_common/exceptions.py

```diff
@@ -72,6 +72,13 @@ def __init__(self, msg: str | None = None):
         super().__init__(msg or "Error while transferring to/from S3 storage")
 
 
+class AWSS3400RequestTimeOutError(NodeportsException):
+    """S3 transfer error"""

```

Comment:
maybe that is where you should put the description and the reference to the error, since this is the error